.github/workflows/test.yml

```diff
@@ -0,0 +1,18 @@
+name: test-hook
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Setup Ruby
+        uses: actions/setup-ruby@v1
+        with:
+          ruby-version: 2.7.x
+      - name: Build and run tests
+        run: |
+          gem install bundler
+          bundle install
+          rspec spec/
```

Comment:
This config file runs `github` action

Reply:
This GitHub action will be convenient for running tests. Thanks.